tests/tests_server.py

```diff
@@ -179,24 +180,23 @@ def test_get_event_from_type_node_BaseEvent(self):
     def test_get_event_from_type_node_Inhereted_AuditEvent(self):
         ev = opcua.common.events.get_event_obj_from_type_node(opcua.Node(self.opc.iserver.isession, ua.NodeId(ua.ObjectIds.AuditEventType)))
         self.assertIsNot(ev, None)  # we did not receive event
-        self.assertIsInstance(ev, ua.BaseEvent)
-        self.assertIsInstance(ev, ua.AuditEvent)
+        self.assertIsInstance(ev, BaseEvent)
+        self.assertIsInstance(ev, AuditEvent)
         self.assertEqual(ev.EventType, ua.NodeId(ua.ObjectIds.AuditEventType))
-        self.assertEqual(ev.Severity, ua.Variant(1, ua.VariantType.UInt16))
```

Comment:
In events some properties were python built-in, other variants, I changed class to only use built-in. Type is stored in self.data_type (yes a bit ugly). We can add test for data type if you think it is important.
Anyway events are defined by constructing event type where data type can be clearly specified